### PR TITLE
Feat/3/shared components

### PR DESCRIPTION
**Pull request by mershalova**
added layout(header footer sidebar themes)

### CODE REVIEW

File: MilitaryManager.Attachments.API/Properties/launchSettings.json [hunk 1]
```diff
@@ -21,17 +21,16 @@
...
     "Docker": {
       "commandName": "Docker",
       "launchBrowser": true,
       "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort}/weatherforecast",
-      "publishAllPorts": true,
-      "useSSL": true
+      "environmentVariables": {}
     }
   }
 }
```
Comment (vladtymo) [inline MilitaryManager.Attachments.API/Properties/launchSettings.json]: Add the line break.

File: MilitaryManager/client-app/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -2,15 +2,28 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { Test1Component } from './components/test1/test1.component';
 import { Test2Component } from './components/test2/test2.component';
+import { HomeComponent } from './components/home/home.component';
+import { AppLayoutComponent } from './layout/app.layout.component';
 
+// const routes: Routes = [
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/app-routing.module.ts]: Remove these comments. We should clean up the code in the PR.

File: MilitaryManager/client-app/src/app/components/home/home.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/home/home.component.spec.ts]: Delete the spec.ts files as we don't use unit tests.

File: docker-compose.override.yml [hunk 4]
```diff
@@ -8,9 +8,10 @@ services:
     ports:
       - "5000:80"
       - "5001:443"
+
     volumes:
-      - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
```
Comment (vladtymo) [inline docker-compose.override.yml]: Revert the configurations to the previous values or configure for MacOS separately.

**Review by vladtymo — CHANGES_REQUESTED** [review]

**Review by vladtymo — APPROVED** [review]

File: MilitaryManager/client-app/src/app/app-routing.module.ts [hunk 5]
```diff
@@ -2,15 +2,20 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { Test1Component } from './components/test1/test1.component';
 import { Test2Component } from './components/test2/test2.component';
+import { HomeComponent } from './components/home/home.component';
+import { AppLayoutComponent } from './layout/app.layout.component';
 
 const routes: Routes = [
-    { path: '', redirectTo: 'test1', pathMatch: 'full' },
+    {path:'', component:AppLayoutComponent,
+  children:[
```
Comment (5haki6) [inline MilitaryManager/client-app/src/app/app-routing.module.ts:10]: bad aligment

**Review by 5haki6 — APPROVED** [review]

**Pull request merged**